docs/sofapython3/tutorials/Tripod/details/tripod.py

```diff
@@ -103,66 +54,115 @@ def __attachToActuatedArms(self, radius=60, numMotors=3, angleShift=180.0):
                        effectorPos[2], 0, 0, 0, 1])
 
         # Rigidify the deformable part at extremity to attach arms
-        rigidifiedstruct = Rigidify(self, deformableObject, groupIndices=groupIndices, frames=frames, name="RigidifiedStructure")
+        rigidifiedstruct = Rigidify(self, deformableObject, groupIndices=groupIndices, frames=frames,
+                                    name="RigidifiedStructure")
 
         # Use this to activate some rendering on the rigidified object ######################################
-        # setData(rigidifiedstruct.RigidParts.dofs, showObject=True, showObjectScale=10, drawMode=2)
-        # setData(rigidifiedstruct.RigidParts.RigidifiedParticules.dofs, showObject=True, showObjectScale=1,
-        #         drawMode=1, showColor=[1., 1., 0., 1.])
-        # setData(rigidifiedstruct.DeformableParts.dofs, showObject=True, showObjectScale=1, drawMode=2)
+        setData(rigidifiedstruct.RigidParts.dofs, showObject=True, showObjectScale=10, drawMode=2)
+        setData(rigidifiedstruct.RigidParts.RigidifiedParticules.dofs, showObject=True, showObjectScale=1,
+                drawMode=1, showColor=[1., 1., 0., 1.])
+        setData(rigidifiedstruct.DeformableParts.dofs, showObject=True, showObjectScale=1, drawMode=2)
```

Comment:
If you agree that this is not necessary, there's another one here.

Reply:
Definitely this is one is the worse